frontend/src/v5/ui/routes/dashboard/projects/federations/editFederationModal/editFederation/editFederation.component.tsx

```diff
@@ -52,7 +53,7 @@ export const EditFederation = ({ federation, onContainersChange }: EditFederatio
 	}, [containers]);
 
 	useEffect(() => {
-		onContainersChange(includedContainers);
+		onContainersChange(pickBy(includedContainers, identity)); // filter out 'undefined' which are unassigned containers
```

Comment:
`pickBy` seems to return an object rather than an array, maybe you could use `includedContainers.filter(Boolean)`?

I think this should also fix the problems I mentioned early in the main feed

Reply:
Should the `unassigned` container be displayed though when a collaborator is editing the federation?